src/hotkey/hotkey_item.cpp

```diff
@@ -157,6 +157,7 @@ bool hotkey_base::matches(const SDL_Event &event) const
 void hotkey_base::save(config& item) const
 {
 	item["command"] = get_command();
+	item["disabled"] = is_disabled() ? config::attribute_value::s_true : config::attribute_value::s_false;
```

Comment:
There was no way I could have known that from the code - what should I use instead, then? Surely not hard-coded true/false/yes/no? It is stored and read from config objects so I thought I should use config attributes instead of just hard-coded values.